### ISSUE TITLE
Performance and crashing on iOS 10

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/energy-skate-park#55]
For https://github.com/phetsims/QA/issues/226.
There is a fair amount of slow down on the measure screen in particular of ESP.  This can be seen in iOS 9.  However, in iOS 10 the sim has a tendency to crash rather quickly, especially when this screen has been or is being used.  I wiped the iOS 10 device as it has had problems in the past.  This seemed to improve other sims.  ESP was still crashing however, so there is a possibility that this is not due to the device itself.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe. The measure screen has extra animation for the circles representing data samples of the skater. Im sure it is related to that as I have since learned that all animation needs to be in the WebGL layer. #42 should be decided before any work goes into this issue.

**Comment by jessegreenberg** [comment]
No longer on hold because the sim is not using WebGL anymore. Not sure if this will still be an issue. @KatieWoe can you please check? Performance of measure screen is great on iOS 12, but I don't have access to iOS 10.

**Comment by jessegreenberg** [comment]
Thanks for checking @KatieWoe I am not surprised that the graph screen is slow at the moment, there is prototype code there. Since the measure screen is OK and no exhibiting any crashing I am going to close this.
